src/core/src/op/eye.cpp

```diff
@@ -24,8 +24,6 @@ bool evaluate_eye(const ov::HostTensorPtr& out, const int64_t diagonal_index) {
     switch (out->get_element_type()) {
         NGRAPH_TYPE_CASE(evaluate, i8, out, diagonal_index);
         NGRAPH_TYPE_CASE(evaluate, u8, out, diagonal_index);
-        NGRAPH_TYPE_CASE(evaluate, f16, out, diagonal_index);
```

Comment:
I agree that looks like we don't need these data types.
But it is not a backward compatible change.
@mitruska Do you remember why we added these data types?

Reply:
I believe it was considered as a default set of data types to be supported by evaluate.
Also the mentioned PR #11538 was a continuation of the implementation proposed in #11240 by @a-sidorova